nomad/structs/structs.go

```diff
@@ -4156,17 +4156,19 @@ const (
 	// JobMinPriority is the minimum allowed priority
 	JobMinPriority = 1
 
-	// JobDefaultPriority is the default priority if not
-	// not specified.
+	// JobDefaultPriority is the default priority if not specified.
 	JobDefaultPriority = 50
 
-	// JobMaxPriority is the maximum allowed priority
-	JobMaxPriority = 100
+	// JobDefaultMaxPriority is the default maximum allowed priority
```

Comment:
Going to nitpick on the comment text here:

```suggestion
	// JobDefaultMaxPriority is the maximum allowed value for the default priority
```